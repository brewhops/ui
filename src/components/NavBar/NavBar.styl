@import '../../styles/breakpoints'

.navbar {
  display flex
  position sticky
  top 0
  background-color #00aaa6
  justify-content center
  z-index 1
}

.option {
  color #ffffff
  text-decoration none
  line-height 50px
  padding 0 10px
  font-size 14pt
  +less-than(tablet){
    font-size 12pt
  }
}

/* onclick color: 267f7d */
.option:hover {
  background-color: #00ccc7;
}
.active {
  background-color: #267f7d;
}