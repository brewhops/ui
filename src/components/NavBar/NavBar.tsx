import React from 'react';
import { Link } from 'react-router-dom';

export const NavBar: React.FC = () => 
<nav className="navbar">
  <Link to="/tank-monitoring">Home</Link>
  <Link to="/batch-history">Batch History</Link>
  <Link to="/tank-history">Tank History</Link>
  <Link to="/admin">Admin</Link>
  <a>Logout</a>
</nav>