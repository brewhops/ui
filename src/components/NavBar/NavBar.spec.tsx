import React from 'react';
import { NavBar } from './NavBar';
import { shallow } from 'enzyme';

describe('src/components/NavBar.tsx', () => {
  it('renders the nav bar', () => {
    expect(shallow(<NavBar/>)).toMatchSnapshot();
  })
});