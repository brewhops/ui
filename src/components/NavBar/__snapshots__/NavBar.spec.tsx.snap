// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/NavBar.tsx renders the nav bar 1`] = `ShallowWrapper {}`;
